src/compas/files/__init__.py

```diff
@@ -13,9 +13,12 @@
     :toctree: generated/
     :nosignatures:
 
+    BaseReader
     OBJ
     OBJReader
     OBJParser
+    OFF
+    OFFReader
```

Comment:
i guess these need to be removed for now

Reply:
Ok!